examples/persistent-storage/esp32/.gitignore

```diff
@@ -0,0 +1,5 @@
+*.vscode
```

Comment:
should this gitignore be at a top level instead inside examples?

Reply:
I could, just copied the style form all-cluster-app/esp